Makefile

```diff
@@ -27,22 +27,28 @@ BUILDFLAGS ?=
 LDFLAGS ?=
 TAGS ?=
 
-# set to 1 for a completely static build. Otherwise if set to 0
-# a dynamic binary is produced that requires glibc to be installed
-STATIC ?= 1
-ifeq ($(STATIC),1)
-TAGS += static
-else
+# if set to 'dynamic' all dependencies are dynamically linked. if
+# set to 'static' all dependencies will be statically linked. If set
+# to 'stdlib' then the standard library will be dynamically
+# linked and everything else will be statically linked.
+LINKMODE ?= dynamic
```

Comment:
its not possible to build static on host machine, libcurl for example on Ubuntu brings in the world. I can look into making static the default when building inside the container but I'm not sure if there's an easy way to do so or if it would be confusing